wandb/sdk/data_types/object_3d.py

```diff
@@ -255,6 +264,17 @@ def from_point_cloud(
         point_cloud_type: "PointCloudType" = "lidar/beta",
         # camera: Optional[Camera] = None,
     ) -> "Object3D":
```

Comment:
It was initially unclear what the types where for points, boxes, and vectors for `from_point_cloud` (e.g. `Sequence['Points']`).

Would it be improper to instead to "array" or list of dictionaries, etc.?